packages/ui5-application-inquirer/src/prompts/validators.ts

```diff
@@ -32,7 +32,11 @@ export function validateAppName(appName: string, targetDir: string): boolean | s
 export async function validateFioriAppProjectFolder(targetDir: string): Promise<string | boolean> {
     const appRoot = await findRootsForPath(targetDir);
     if (appRoot) {
-        return t('validators.folderContainsFioriApp', { path: appRoot.appRoot });
+        return t('validators.folderContainsFioriApp', { path: appRoot?.appRoot });
```

Comment:
I don't think there's any need for the `?` in `{ path: appRoot?.appRoot }` becuase you already check if it's defined

Reply:
Thank you for the review. Removed as suggested.